### ISSUE TITLE
inline warnings

### ISSUE THREAD

**Issue by telephon** [scztt/Connection.quark#3]
There are a number of inline warnings (try switching on inline warnings to see them):

```
WARNING: FunctionDef contains variable declarations and so will not be inlined.
  in file '~/Library/Application Support/SuperCollider/downloaded-quarks/Connection/Connection.sc'
  line 62 char 30:

  			^object.dependants !? { |d| d.includes(dependant) } ?? false;
                                
  		}
-----------------------------------
WARNING: FunctionDef contains variable declarations and so will not be inlined.
  in file '~/Library/Application Support/SuperCollider/downloaded-quarks/Connection/Connection.sc'
  line 141 char 36:

  				var tempTrace = traceConnection;
                                      
  				traceConnection = nil;
-----------------------------------
WARNING: FunctionDef contains variable declarations and so will not be inlined.
  in file '~/Library/Application Support/SuperCollider/downloaded-quarks/Singleton/Singleton.sc'
  line 75 char 40:

  			var key = dict.findKeyForValue(sing);
                                          
  			if (key.notNil) {
-----------------------------------
WARNING: FunctionDef contains variable declarations and so will not be inlined.
  in file '~/Library/Application Support/SuperCollider/downloaded-quarks/Connection/UpdateDispatcher.sc'
  line 19 char 0:

  			|d|
  
  			d.free;
-----------------------------------
WARNING: FunctionDef contains variable declarations and so will not be inlined.
  in file '~/Library/Application Support/SuperCollider/downloaded-quarks/Connection/UpdateDispatcher.sc'
  line 76 char 0:

  			|item|
  
  			item.update(obj, changed, *args);
-----------------------------------
WARNING: FunctionDef contains variable declarations and so will not be inlined.
  in file '~/Library/Application Support/SuperCollider/downloaded-quarks/Connection/UpdateDispatcher.sc'
  line 83 char 33:

  		^dispatchTable[obj] !? { |conn| conn.dependant.connectionTraceString(obj, what) } ?? { "%(%) - no target".format(this.class, this.identityHash) }
                                   
  	}
-----------------------------------
WARNING: FunctionDef contains variable declarations and so will not be inlined.
  in file '~/Library/Application Support/SuperCollider/downloaded-quarks/Connection/UpdateForwarder.sc'
  line 196 char 44:

  					var tmpdeferredUpdate = deferredUpdate;
                                              
  					deferredUpdate = nil;
-----------------------------------
WARNING: FunctionDef contains variable declarations and so will not be inlined.
  in file '~/Library/Application Support/SuperCollider/downloaded-quarks/Connection/extDictionary.sc'
  line 8 char 36:

  			var newVal = defaultFunc.value();
                                      
  			this.put(key, newVal);
-----------------------------------
```

**Comment by scztt** [comment]
Fixed by a0e29fe0aa2f3c4440fdf07d0474541dcb8fe3d6. Thanks @telephon.

**Comment by telephon** [comment]
thanks!
